packages/lexicon-resolver/src/client/index.ts

```diff
@@ -0,0 +1,67 @@
+/**
```

Comment:
You should also add these files to `.gitattributes` so they get ignored in Guthub diffs.